public/js/monitor.js

```diff
@@ -390,27 +415,28 @@ function handleFormSubmits(formEvent) {
     formEvent.target.classList.add("loading-data");
     addUser(formEvent);
     setModalTabbing();
-    ga_sendPing("SignUp_Complete", false);
+    sendPing(formEvent.target, "Success");
+    ga_sendLegacyPing("SignUp_Complete", false);
     return;
   }
   formEvent.submit();
   return;
 }
 
-function doButtonRouting(event) {
+async function doButtonRouting(event) {
```

Comment:
Does this need to be `async` ? You're calling async functions inside here, but you're not `await`ing for any of them.